packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -755,9 +755,11 @@ Specifies a set of Groups whose Users are to be included or excluded
 
 Specifies an authentication entry point
 
-| Parameter | Description | Data Type         | Required | Default |
-| --------- | ----------- | ----------------- | -------- | ------- |
-| authType  |             | `ANY` or `RADIUS` | No       |         |
+| Parameter | Description                              | Data Type                             | Required | Default |
+| --------- | ---------------------------------------- | ------------------------------------- | -------- | ------- |
+| authType  | This tells how the user is authenticated | `ANY` or `RADIUS` or `LDAP_INTERFACE` | No       |  `ANY`  |
+
+`LDAP_INTERFACE` is an <ApiLifecycle access="ea" /> release.
```

Comment:
I think this statement is confusing. `LDAP_INTERFACE` is not a release. My suggestion:
Note: The `LDAP_INTERFACE` data type option is an <ApiLifecycle access="ea" /> feature.

also nit: I think it'd be cleaner to put the statement in a note block, something like this:
![Screen Shot 2020-12-08 at 11 57 04 AM](https://user-images.githubusercontent.com/12566503/101534859-9aa90d00-394c-11eb-8e6f-328f8edaa19d.png)